src/transformers/models/auto/tokenization_auto.py

```diff
@@ -203,6 +203,7 @@
                     "GemmaTokenizerFast" if is_tokenizers_available() else None,
                 ),
             ),
+            ("modernbert", (None, "GPTNeoXTokenizerFast" if is_tokenizers_available() else None)),
```

Comment:
```suggestion
            ("modernbert", (None, "PreTrainedTokenizerFast" if is_tokenizers_available() else None)),
```
`GPTNeoXTokenizerFast` differs from what was used during training.
We could also use`ModernBertTokenizerFast` depending on my previous comment.

Reply:
For the moment, I have it set as `PreTrainedTokenizerFast` pending discussion.